locales/en.json

```diff
@@ -253,6 +253,7 @@
     "certmanager_domain_http_not_working": "It seems that the domain {domain:s} cannot be accessed through HTTP. Please check your DNS and nginx configuration is okay",
     "certmanager_error_no_A_record": "No DNS 'A' record found for {domain:s}. You need to make your domain name point to your machine to be able to install a Let's Encrypt certificate! (If you know what you are doing, use --no-checks to disable those checks.)",
     "certmanager_domain_dns_ip_differs_from_public_ip": "The DNS 'A' record for domain {domain:s} is different from this server IP. If you recently modified your A record, please wait for it to propagate (some DNS propagation checkers are available online). (If you know what you are doing, use --no-checks to disable those checks.)",
+    "certmanager_domain_not_resolved_locally": "The domain {domain:s} cannot be resolved locally. This might happen if you recently modified your DNS record. If so, please wait a few hours for it to propagate. If the issue persists, consider adding {domain:s} to /etc/hosts. (If you know what you are doing, use --no-checks to disable those checks.)",
```

Comment:
> Also I don't agree with the interpretation of /etc/resolv.conf. To me there's no right or wrong resolv.conf, and even if you use "nameserver 127.0.0.1" (such that it uses the local dnsmasq service ?), "toto.nohost.me" won't resolve if it's not in /etc/hosts... (or if we tweak dnsmasq in some other ways, but to me /etc/hosts is the most reliable :/)

It will resolve, try doing a `yunohost domain add pouet.pouet` then "ping pouet.pouet" and you'll get an ip for `pouet.pouet`.

Interestingly it doesn't resolve other domains and I don't know why.